data/packages/com.dopamine.firebase.core.yml

```diff
@@ -1,6 +1,6 @@
 name: com.dopamine.firebase.core
 displayName: Firebase Core
-description: Latest Version of firebase
+description: Firebase core version 7.0.2
```

Comment:
Hi @dopaminegamesstudio,

If you leave the version information in the description, then every time the package gets a major update the description will be outdated. Maybe just "Firebase core component"?